mmv1/templates/terraform/resource.erb

```diff
@@ -63,6 +63,12 @@ import (
     timeouts ||= Api::Timeouts.new
 -%>
 
+<%  if object.async&.is_a? Api::OpAsync -%>
+    <%  if object.async.include_project -%>
```

Comment:
Is this required? It seems like `object.async` isn't set. It would be great to revert this change as well if it isn't used by this change.

Reply:
This is causing the lint failures:

```
google-beta/resource_vertex_ai_featurestore_entitytype_feature.go:28:5: project redeclared in this block
	google-beta/resource_vertex_ai_featurestore_entitytype.go:28:5: other declaration of project
```

Reply:
bad rebase, checked out file from main